pipelines/internal/commands/run/run.go

```diff
@@ -180,30 +181,55 @@ func Invoke(ctx context.Context, service *genomics.Service, project string, argu
 	}
 	fmt.Printf("%s\n", encoded)
 
+	if *attempts < 1 {
+		return fmt.Errorf("The number of attempts must be > 1")
+	}
+
 	if *dryRun {
 		return nil
 	}
 
-	lro, err := service.Pipelines.Run(req).Context(ctx).Do()
-	if err != nil {
-		if err, ok := err.(*googleapi.Error); ok {
-			return fmt.Errorf("starting pipeline: %q: %q", err.Message, err.Body)
+	attempt:=uint(1)
```

Comment:
You need to run your source files through gofmt to correct issues like this (attempt := uint(1))